src/integrationTest/docker/docker-compose.yml

```diff
@@ -0,0 +1,173 @@
+---
+version: '2.1'
+
+services:
+  #---------------------------------------------------------------------------#
+  # Zookeeper Cluster                                                         #
+  #---------------------------------------------------------------------------#
+  zookeeper-1:
+    image: confluentinc/cp-zookeeper:4.1.0
```

Comment:
Is `confluentinc/cp-zookeeper:4.1.0` compatible with `confluentinc/cp-kafka-connect-base:5.0.0`?

Reply:
Yes works fine. I am using the same in production.